clients/src/main/java/org/apache/kafka/common/errors/GroupSubscribedToTopicException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.errors;
+
+public class GroupSubscribedToTopicException extends ApiException {
```

Comment:
This is a public interface and wasn't mentioned in the KIP. Was there an offline discussion on adding this?

Reply:
Right. There was not an offline discussion about it. I have forgotten to bring the point. What's the best way? Shall I discuss thing in the voting thread?

To give the context: When one tried to delete the offsets of an active consumer group, we need a way to indicate if the topic is still actively consumed by the group which means that it is not possible to delete the offsets.

Reply:
@hachikuji What do you think about adding this one?

Reply:
I think this makes sense, and you can bring it up in the vote thread and ask if anyone objects.

Reply:
Good catch @stanislavkozlovski :)

Reply:
Seems reasonable to me.